notebook/automl_time_series_forecast.ipynb

```diff
@@ -1915,7 +1761,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
+   "execution_count": 21,
```

Comment:
The result here is not so good, both in terms of mape and the visualization. Is it because the series is hard to predict?